### ISSUE TITLE
The `FindPythonInterp` will be removed by cmake

### ISSUE THREAD

**Issue by shushanhf** [dotnet/runtime#89424]
### Description

There are some `CMakeLists.txt` which using the `FindPythonInterp`, but the `FindPythonInterp` removed by cmake.

https://cmake.org/cmake/help/v3.27/module/FindPythonInterp.html
https://cmake.org/cmake/help/v3.27/policy/CMP0148.html#policy:CMP0148

### Reproduction Steps

building the runtime by newest OS which the CMake version is 3.27.
There will be some warings about CMake.

### Expected behavior

no these warnings.

### Actual behavior

```
CMake Warning (dev) at vm/eventing/CMakeLists.txt:11 (include):
  Policy CMP0148 is not set: The FindPythonInterp and FindPythonLibs modules
  are removed.  Run "cmake --help-policy CMP0148" for policy details.  Use 
  the cmake_policy command to set the policy and suppress this warning.

This warning is for project developers.  Use -Wno-dev to suppress it. 

CMake Warning (dev) at vm/eventing/eventpipe/CMakeLists.txt:1 (include):
  Policy CMP0148 is not set: The FindPythonInterp and FindPythonLibs modules
  are removed.  Run "cmake --help-policy CMP0148" for policy details.  Use 
  the cmake_policy command to set the policy and suppress this warning.

This warning is for project developers.  Use -Wno-dev to suppress it. 
```

**Comment by jkoritzinsky** [comment]
Fixed by https://github.com/dotnet/runtime/pull/89546

**Comment by shushanhf** [comment]
> Fixed by #89546

How about the file  `src/coreclr/pal/src/eventprovider/lttngprovider/CMakeLists.txt`  https://github.com/dotnet/runtime/blob/242242947c01615b243e8b4d9ee163030a4ceb47/src/coreclr/pal/src/eventprovider/lttngprovider/CMakeLists.txt#L1？
Here is still using the `FindPythonInterp`.
